docs/advanced-features/using-mdx.md

```diff
@@ -0,0 +1,166 @@
+---
+description: Learn how to use @next/mdx in your Next.js project.
+---
+
+# Using MDX with Next.js
+
+Next.js supports MDX through a number of different means, this page will outline some of the ways you can begin integrating MDX into your Next.js project.
+
+## What is MDX?
+
+MDX is a superset of markdown that lets you write JSX directly in your markdown files. It is a powerful way to add dynamic interactivity, and embed components within your content, helping you to bring your pages to life.
+
+### MDX plugins
```

Comment:
Title case, please! We should be consistent until a larger change is made.

Reply:
We are moving away from title case! We can't create 1 massive PR which changes them all. It has to be incremental.

Reply:
Im of the opinion that we should change to sentence case. However, I'll make the change to title case here, but create an issue to be tackled later for sentence case.